wodles/aws/constants.py

```diff
@@ -0,0 +1,105 @@
+DEFAULT_AWS_BUCKET_DATABASE_NAME = "s3_cloudtrail"
+
+
+# DATABASE QUERIES
+SQL_COUNT_ROWS = """SELECT count(*) FROM {table_name};"""
+
+# REGIONS
+ALL_REGIONS = (
+    'af-south-1', 'ap-east-1', 'ap-northeast-1', 'ap-northeast-2', 'ap-northeast-3', 'ap-south-1', 'ap-south-2',
+    'ap-southeast-1', 'ap-southeast-2', 'ap-southeast-3', 'ap-southeast-4', 'ca-central-1', 'eu-central-1',
+    'eu-central-2', 'eu-north-1', 'eu-south-1', 'eu-south-2', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'il-central-1',
+    'me-central-1', 'me-south-1', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'
+)
+DEFAULT_GOV_REGIONS = {'us-gov-east-1', 'us-gov-west-1'}
+SERVICES_REQUIRING_REGION = {'inspector', 'cloudwatchlogs'}
+
+
+# URLS
+RETRY_CONFIGURATION_URL = 'https://documentation.wazuh.com/current/amazon/services/prerequisites/' \
+                          'considerations.html#Connection-configuration-for-retries'
+GUARDDUTY_URL = 'https://documentation.wazuh.com/current/amazon/services/supported-services/guardduty.html'
+SECURITY_LAKE_IAM_ROLE_AUTHENTICATION_URL = 'https://documentation.wazuh.com/current/cloud-security/amazon/services/' \
+                                        'supported-services/security-lake.html#configuring-an-iam-role'
+
+# PATHS
+DEFAULT_AWS_CONFIG_PATH = path.join(path.expanduser('~'), '.aws', 'config')
+WODLES_PATH = 'wodles/aws'
+QUEUE_PATH = 'queue/sockets/queue'
+
+# MSG HEADERS
+WAZUH_AWS_MESSAGE_HEADER = "1:Wazuh-AWS:"
+
+# MSG TEMPLATES
+AWS_BUCKET_MSG_TEMPLATE = {'integration': 'aws',
+                           'aws': {'log_info': {'aws_account_alias': '', 'log_file': '', 's3bucket': ''}}}
+AWS_SERVICE_MSG_TEMPLATE = {'integration': 'aws', 'aws': ''}
+
+# AWS INTEGRATION DEPRECATED VALUES
+DEPRECATED_AWS_INTEGRATION_TABLES = {'log_progress', 'trail_progress'}
+
+# DEPRECATED MESSAGES
+GUARDDUTY_DEPRECATED_MESSAGE = 'The functionality to process GuardDuty logs stored in S3 via Kinesis was deprecated ' \
+                               'in {release}. Consider configuring GuardDuty to store its findings directly in an S3 ' \
+                               'bucket instead. Check {url} for more information.'
+
+# ERROR NAMES
+INVALID_CREDENTIALS_ERROR_NAME = "SignatureDoesNotMatch"
+INVALID_REQUEST_TIME_ERROR_NAME = "RequestTimeTooSkewed"
+THROTTLING_EXCEPTION_ERROR_NAME = "ThrottlingException"
+
+# ERROR CODES
+UNKNOWN_ERROR_CODE = 1
+INVALID_CREDENTIALS_ERROR_CODE = 3
+METADATA_ERROR_CODE = 5
+UNABLE_TO_CREATE_DB = 6
+UNEXPECTED_ERROR_WORKING_WITH_S3 = 7
+DECOMPRESS_FILE_ERROR_CODE = 8
+PARSE_FILE_ERROR_CODE = 9
+UNABLE_TO_CONNECT_SOCKET_ERROR_CODE = 11
+INVALID_TYPE_ERROR_CODE = 12
+SENDING_MESSAGE_SOCKET_ERROR_CODE = 13
+EMPTY_BUCKET_ERROR_CODE = 14
+INVALID_ENDPOINT_ERROR_CODE = 15
+THROTTLING_ERROR_CODE = 16
+INVALID_KEY_FORMAT_ERROR_CODE = 17
+INVALID_PREFIX_ERROR_CODE = 18
+INVALID_REQUEST_TIME_ERROR_CODE = 19
+UNABLE_TO_FETCH_DELETE_FROM_QUEUE = 21
+INVALID_REGION_ERROR_CODE = 22
```

Comment:
These constants are not used in the source code but only in the unit tests. Please, move them to the `tests/aws_constants.py` file.

Reply:
Changed in c223f95e6b2e95be79cdfeb5087d9c916fef59ad.

<details>
<summary>UT results after changes</summary>

```console
========================================================================================================== test session starts ==========================================================================================================
platform linux -- Python 3.10.12, pytest-7.3.1, pluggy-1.5.0
rootdir: /home/federamos/Documents/Wazuh/Repositories/wazuh/wodles/aws/tests
configfile: pytest.ini
plugins: asyncio-0.18.1, html-2.1.1, metadata-3.1.1, cov-4.1.0, tavern-1.23.5, anyio-4.1.0, trio-0.8.0
asyncio: mode=auto
collected 599 items                                                                                                                                                                                                                     

wodles/aws/tests/test_aws_bucket.py ............................................................................................................................................................................................. [ 31%]
..............                                                                                                                                                                                                                    [ 33%]
wodles/aws/tests/test_aws_s3.py ......................                                                                                                                                                                            [ 37%]
wodles/aws/tests/test_aws_service.py ....                                                                                                                                                                                         [ 38%]
wodles/aws/tests/test_cloudtrail.py ..                                                                                                                                                                                            [ 38%]
wodles/aws/tests/test_cloudwatchlogs.py .....................................................                                                                                                                                     [ 47%]
wodles/aws/tests/test_config.py ..............................................................................                                                                                                                    [ 60%]
wodles/aws/tests/test_guardduty.py .................                                                                                                                                                                              [ 63%]
wodles/aws/tests/test_inspector.py ......                                                                                                                                                                                         [ 64%]
wodles/aws/tests/test_load_balancers.py ............                                                                                                                                                                              [ 66%]
wodles/aws/tests/test_s3_log_handler.py .......................                                                                                                                                                                   [ 70%]
wodles/aws/tests/test_server_access.py .................................                                                                                                                                                          [ 75%]
wodles/aws/tests/test_sqs_message_processor.py ........                                                                                                                                                                           [ 76%]
wodles/aws/tests/test_sqs_queue.py .......                                                                                                                                                                                        [ 78%]
wodles/aws/tests/test_tools.py ..................................                                                                                                                                                                 [ 83%]
wodles/aws/tests/test_umbrella.py ......                                                                                                                                                                                          [ 84%]
wodles/aws/tests/test_vpcflow.py .....................                                                                                                                                                                            [ 88%]
wodles/aws/tests/test_waf.py .......                                                                                                                                                                                              [ 89%]
wodles/aws/tests/test_wazuh_integration.py ...............................................................                                                                                                                        [100%]

========================================================================================================== 599 passed in 3.35s ==========================================================================================================
```

</details>